src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -33,6 +33,20 @@ namespace picongpu
 namespace traits
 {
     template<>
+    struct PICToAdios<bool>
+    {
+        ADIOS_DATATYPES type;
+
+        PICToAdios() :
+        type(adios_unsigned_byte) {}
```

Comment:
I assume `ADIOS_DATATYPES` is  a type for data-types - is this correct?
If so could you not write something like 
```c++
constexpr ADIOS_DATATYPES type = adios_unsigned_byte;

PICToAdios() { }
```